packages/components/src/focal-point-picker/index.native.js

```diff
@@ -120,7 +120,7 @@ function FocalPointPicker( props ) {
 					setSliderKey( ( prevState ) => prevState + 1 );
 				},
 			} ),
-		[ containerSize ]
+		[ containerSize, pan, onChange, shouldEnableBottomSheetScroll ]
```

Comment:
It looks like `onChange` is changing on every change that's happening on the Focal point picker when updating the sliders and during the touch events, making the `panResponder` to be created constantly.

This prop is coming from [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/focal-point-settings-panel/index.native.js#L59). So I think we should add a `useCallback` there (for `setPosition`) to prevent this from happening.

Reply:
Thanks @geriux! Let me know if [e963825](https://github.com/WordPress/gutenberg/pull/41520/commits/e963825fdaa37791d9163c7ce29809058ebb1ec4) helps improve things! 😄

Reply:
Actually scratch that. There's a typo in this commit that was preventing the linter from detecting further issues. Stand by! 😬

Reply:
> It looks like `onChange` is changing on every change that's happening on the Focal point picker when updating the sliders and during the touch events, making the `panResponder` to be created constantly.

@geriux good catch! How did you come across the changing `onChange` value? Did you notice a performance issue or observe it in a different manner? Primarily curious to improve my ability to identify this type of thing in the future.